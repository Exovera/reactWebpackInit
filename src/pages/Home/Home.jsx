import React from 'react';
import PropTypes from 'prop-types';

const Home = () => {
	return (
		<div className="home">
			Home sweet home
		</div>
	);
};


Home.propTypes = {

};


export default Home;
