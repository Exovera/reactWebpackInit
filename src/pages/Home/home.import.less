.home {
	background-color: @off-white-1;
}