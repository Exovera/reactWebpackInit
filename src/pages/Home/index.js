import HomeContainer from './HomeContainer.js';
import Home from './Home.jsx';

export { Home };
export default HomeContainer;