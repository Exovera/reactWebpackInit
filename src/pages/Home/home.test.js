import React from "react";
import Enzyme, { mount } from "enzyme";
import toJSON from "enzyme-to-json";
import Adapter from "enzyme-adapter-react-16";
import Home from "./Home.jsx";

Enzyme.configure({ adapter: new Adapter() });

function setup(passedProps = {}) {
	const props = {
		activeNarrative: passedProps.activeNarrative || "all",
		// activeNarrativeData: passedProps.activeNarrativeData || null,
		// filtersDisplayText: passedProps.filtersDisplayText || false,
		// scrollbarSize: passedProps.scrollPosition || 15,
		// scrollPosition: passedProps.scrollPosition || 0,
	};

	const enzymeWrapper = mount(<Home {...props} />);

	return {
		props,
		enzymeWrapper,
	};
}

describe("Home page", () => {
	it('should render self.', () => {
		const { enzymeWrapper } = setup();
		expect(toJSON(enzymeWrapper)).toMatchSnapshot();
	});
});