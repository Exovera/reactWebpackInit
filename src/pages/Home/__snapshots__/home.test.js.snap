// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page should render self. 1`] = `
<Home
  activeNarrative="all"
>
  <div
    className="home"
  >
    Home sweet home
  </div>
</Home>
`;
